argilla-server/src/argilla_server/contexts/accounts.py

```diff
@@ -157,6 +158,11 @@ async def create_user_with_random_password(
     return await create_user(db, user_attrs, workspaces)
 
 
+async def update_user(db: AsyncSession, user: User, user_update: UserUpdate) -> User:
+    params = user_update.dict(exclude_unset=True)
+    return await user.update(db, **params)
```

Comment:
We're moving away from using schemas on contexts files so receiving a dictionary with the attributes would be a good idea:

```suggestion
async def update_user(db: AsyncSession, user: User, user_attrs: dict) -> User:
    return await user.update(db, **user_attrs)
```

You can move the `user_update.dict(exclude_unset=True)` to pass the `user_attrs` to the handler.

Reply:
All good 👍 You've gotten the validation already when hitting the API so yeah this change makes sense to just pass around the dict